examples/animation/README.md

```diff
@@ -0,0 +1,3 @@
+```sh
+cargo run -p animation
+```
```

Comment:
I think, that readme should contain a little more information about what that example is doing and what problem it solves